tracing-opentelemetry/src/layer.rs

```diff
@@ -415,6 +415,32 @@ where
         }
     }
 
+    fn on_follows_from(&self, id: &Id, follows: &Id, ctx: Context<S>) {
+        let span = ctx
+            .span(id)
+            .expect("Span to follow not found, this is a bug");
```

Comment:
nit: `Span not found...`, rather than `Span to follow not found..`

Reply:
Missed that when copy / pasting. Good catch!